travis.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -x # expand and echo commands
+export SPHERE_TRAVIS=1
+
+echo TRAVIS_BUILD_DIR is $TRAVIS_BUILD_DIR
+echo TRAVIS_OS_NAME is $TRAVIS_OS_NAME
+
+if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
+    export HOMEBREW_NO_AUTO_UPDATE=1
+    brew install ninja
+    brew install mysql-client
+    gcc -v && g++ -v && cmake --version
+    echo Building Release
+    mkdir -p release && cd release && cmake -G Ninja ../src -DCMAKE_BUILD_TYPE=Release $EXTRA && cmake --build . || exit 1
```

Comment:
We had tons of problems with Travis-CI trying to upload stuff to the ftp server, or even doing apt-get updates with a sudo-enabled environment. Can you make a similar section in the AppVeyor script?
Also, can you switch from a Release to an automated Nightly build and split the build commands so that we have only one command per line? Imho we have better output in case of an error.
Thanks!